include/pstore/mcrepo/linked_definitions_section.hpp

```diff
@@ -233,16 +238,16 @@ namespace pstore {
 
         private:
             PSTORE_NO_RETURN void error () const;
-            dependents const & d_;
+            linked_definitions const & d_;
         };
 
 
         template <>
-        struct section_to_dispatcher<dependents> {
+        struct section_to_dispatcher<linked_definitions> {
             using type = dependents_dispatcher;
         };
 
     } // end namespace repo
 } // end namespace pstore
 
-#endif // PSTORE_MCREPO_DEPENDENTS_SECTION_HPP
+#endif // PSTORE_MCREPO_LINKED_DEFINITIONS_SECTION_HPP
```

Comment:
The linked_definitions class comments might need to be updated as well. For example, line 128, 132, 136, 138 .. in this file (linked_definitions_section.hpp).

Reply:
Yes, well spotted. I've updated the change with more renaming.